### DISCUSSION TITLE
Ability to duplicate events from list view.

### DISCUSSION THREAD

**Discussion by mike-moreau** [solspace/craft-calendar#235] [Feature Requests]
Craft entries can be duplicated from list view, but that action is not available for events.

It would be a nice time saver if events could be duplicated as well. Thanks for considering.

<img width="1220" alt="Screenshot 2023-07-20 at 2 56 54 PM" src="https://github.com/solspace/craft-calendar/assets/92817780/996f1a33-e191-4ce4-bd93-ec884133487f">

<img width="1130" alt="Screenshot 2023-07-20 at 3 18 23 PM" src="https://github.com/solspace/craft-calendar/assets/92817780/033842f8-f045-4bf5-b1eb-ebdfc89b8b1d">

**Comment by theyve** [comment]
I'm very surprised this isn't already a feature

**Comment by sgtpenguin** [comment]
Just came here thinking the same thing.  This would be a great addition.
